test/menuFactory.spec.js

```diff
@@ -413,5 +413,30 @@ describe('menuFactory', () => {
       ReactDOM.render(<Menu isOpen />, container);
       expect(component.state.isOpen).to.be.true;
     });
+
+    it('will not change when isOpen prop was not changed', () => {
```

Comment:
I'd make the description a little simpler to avoid confusion, maybe something like: 'will always obey isOpen prop'